diff -urN ./waf-1.5.4.orig/waf-light ./waf-1.5.4/waf-light
--- waf-1.5.4.orig/waf-light	2009-04-07 20:34:22.000000000 +0200
+++ waf-1.5.4/waf-light	2009-05-31 13:33:38.703600164 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 # encoding: utf-8
 # Thomas Nagy, 2005-2009
 
@@ -117,7 +117,7 @@
 		if w: return w
 		err("waf-light requires wafadmin -> export WAFDIR=/folder")
 
-	dir = "/lib/%s-%s-%s/" % (WAF, VERSION, REVISION)
+	dir = "/lib/waf/"
 	for i in [INSTALL,'/usr','/usr/local','/opt']:
 		w = test(i+dir)
 		if w: return w
diff -urN ./waf-1.5.4.orig/wscript ./waf-1.5.4/wscript
--- waf-1.5.4.orig/wscript	2009-04-07 20:34:22.000000000 +0200
+++ waf-1.5.4/wscript	2009-05-31 13:35:21.368995500 +0200
@@ -314,7 +314,7 @@
 		if val != True and val != "y": sys.exit(1)
 		create_waf()
 
-	dir = os.path.join('lib', 'waf-%s-%s' % (VERSION, REVISION), 'wafadmin')
+	dir = os.path.join('lib', 'waf', 'wafadmin')
 
 	wafadmin = bld.new_task_gen('py')
 	wafadmin.find_sources_in_dirs('wafadmin', exts=['.py'])
