--- waf-1.6.4/waf-light.orig	2011-04-17 12:02:50.000000000 +0200
+++ waf-1.6.4/waf-light	2011-05-01 06:55:29.260143094 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 # encoding: ISO8859-1
 # Thomas Nagy, 2005-2010
 
@@ -34,7 +34,7 @@
 REVISION="x"
 GIT="x"
-INSTALL="x"
+INSTALL="/usr/share"
 C1='x'
 C2='x'
 C3='x'
 cwd = os.getcwd()
@@ -140,7 +140,7 @@ def find_lib():
 		if w: return w
 		err('waf-light requires waflib -> export WAFDIR=/folder')
 
-	dirname = '%s-%s-%s' % (WAF, VERSION, REVISION)
-	for i in (INSTALL,'/usr','/usr/local','/opt'):
-		w = test(i + '/lib/' + dirname)
+	dirname = WAF
+	for i in (INSTALL,'/usr/lib','/usr/local/lib','/opt'):
+		w = test(i + '/' + dirname)
 		if w: return w
--- waf-1.6.4/wscript.orig	2011-04-17 12:02:50.000000000 +0200
+++ waf-1.6.4/wscript	2011-05-01 07:00:51.030153869 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/python
 # encoding: utf-8
 # Thomas Nagy, 2005-2010
 
